docs/usage/shoot_access.md

```diff
@@ -11,11 +11,11 @@ The username associated with such `kubeconfig` will be the same which is used fo
 In order to request such a `kubeconfig`, you can run the following commands:
 
 ```bash
-export NAMESPACE=my-namespace
+export PROJECT_NAME=my-project
 export SHOOT_NAME=my-shoot
 kubectl create \
     -f <path>/<to>/kubeconfig-request.json \
-    --raw /apis/core.gardener.cloud/v1beta1/namespaces/${NAMESPACE}/shoots/${SHOOT_NAME}/adminkubeconfig | jq -r ".status.kubeconfig" | base64 -d
+    --raw /apis/core.gardener.cloud/v1beta1/namespaces/garden-${PROJECT_NAME}/shoots/${SHOOT_NAME}/adminkubeconfig | jq -r ".status.kubeconfig" | base64 -d
```

Comment:
The namespace need not be `garden-<project-name>` always, It's possible to have a custom namespace which starts with `garden-`, See https://github.com/gardener/gardener/blob/d2584327576f49a5f705b1e80fda97481f069b26/example/05-project-dev.yaml#L29-L33

Reply:
>  does this mean that "namespace" was set this way when we created them in the shoot yaml?

Not the shoot, it might have been set in your project.
For creating the shoot, the namespace should already be present.

Reply:
That makes sense.  I see namespace on our shoot yaml files.   I did not create our projects so this dot was not connected in my mind.

Thanks for the clarification and pointer.

In this case, I think the PR should be closed unless you disagree.

Reply:
No disagreements.
/close